### DISCUSSION TITLE
Cannot find v22.3 on yum ?

### DISCUSSION THREAD

**Discussion by mayank-pant** [ClickHouse/ClickHouse#35426] [Q&A] [answered]
v22.3 is released but yum update is not showing the latest version of the clickhouse. Pls let me know when will yum repo will be updated with latest clickhouse version.

**Comment by Felixoid** [comment, marked as answer]
Do you use the new repo? https://clickhouse.com/docs/en/getting-started/install/#from-rpm-packages

**UPD**: I recognized that index wasn't updated. I updated it manually and will take a close look on the next update
